src/models/embeddingsList.ts

```diff
@@ -25,14 +25,14 @@ export class EmbeddingsList
   extends Array<Embedding>
   implements OpenAPIEmbeddingsList
 {
-  model: string;
+  model?: string;
   data: Array<Embedding>;
-  usage: EmbeddingsListUsage;
+  usage?: EmbeddingsListUsage;
 
   constructor(
-    model: string,
-    data: Array<Embedding>,
-    usage: EmbeddingsListUsage
+    model?: string,
```

Comment:
Should these be optional? I think you said they're mandatory in the actual API, but do we want ppl to be able to use the endpoint without these? Maybe we use a default model on the backend now (I don't think we did before, but maybe!)

Reply:
These fields aren't enforcing how people call the API, they're just enforcing what `EmbeddingsList` the class is expecting the response to look like, so it should actually be somewhat aligned with what the API spec defines.

When making `embed` requests things are enforced through this type: https://github.com/pinecone-io/pinecone-ts-client/blob/release-candidate/2024-07/src/inference/inference.ts#L32-L36